src/cco-modules/EventOntology.ttl

```diff
@@ -587,6 +587,12 @@ cco:ActOfIssuingMassMediaPressRelease rdf:type owl:Class ;
 ###  http://www.ontologyrepository.com/CommonCoreOntologies/ActOfLegalInstrumentUse
 cco:ActOfLegalInstrumentUse rdf:type owl:Class ;
                             rdfs:subClassOf cco:ActOfArtifactEmployment ;
+                            owl:equivalentClass [ owl:intersectionOf ( [ a owl:Restriction ;
```

Comment:
@justin2004 This axiom is too strong. It implies that any act performed by an agent that involves a legal instrument as a participant is an act of legal instrument use. The axiom does not imply that the agent actually needs to be using the participating legal instrument in the process, which seems to be required. From this follow counter examples: A scholar citing a legal instrument in an argument would seem to satisfy these axioms but not involve an act of legal instrument use.

Reply:
hey @neilotte , good catch.
i've addressed that here:
<img width="306" alt="image" src="https://github.com/user-attachments/assets/f8e37854-8c2f-4d8f-96a8-f259e1310e86">

now a `cco:ActOfLegalInstrumentUse` is a subclass (more specialized) of an act with an agent and a LegalInstrument is a participant.